gst_msdk/gst-libs/mfx/gstmfxfilter.c

```diff
@@ -888,14 +888,20 @@ gst_mfx_filter_set_deinterlace_mode (GstMfxFilter * filter,
   mfxExtVPPDeinterlacing *ext_deinterlacing;
 
   g_return_val_if_fail (filter != NULL, FALSE);
+
+#if MSDK_CHECK_VERSION(1,19)
```

Comment:
Sorry, my bad... yes it means the same thing. Oversight :)
Just fix the indentation here and remove the gstvc1parse.h warning macros. I've tested your meson script, works :)